src/js/components/Chart/stories/Animate.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import { Grommet, Box, Chart, Heading } from 'grommet';
+import { grommet } from 'grommet/themes';
+
+const values = [{ value: [10, 20] }, { value: [20, 30] }, { value: [30, 15] }];
+
+const Example = () => (
+  <Grommet theme={grommet}>
+    <Box direction="row-responsive" wrap pad="large">
+      {['bar', 'line', 'area', 'point'].map(type => (
+        <Box key={type} margin="medium">
+          <Heading size="small" textAlign="center">
+            {type}
+          </Heading>
+          <Chart type={type} values={values} animate />
+        </Box>
+      ))}
+    </Box>
+  </Grommet>
+);
+
+storiesOf('Chart', module).add('Animate', () => <Example />);
```

Comment:
I think we should exclude animated stories from chromatic, the snapshot isn't being captured on time and the Chart isn't included in the snapshot.
Here is how the snapshot is taken (without the Chart):
![image](https://user-images.githubusercontent.com/6320236/90470050-ad66b000-e0d7-11ea-9b3a-8b7c7cb85935.png)

Reply:
fixed